app/src/main/kotlin/org/mifos/mobile/cn/data/models/beneficiaries/listBeneficiaries.kt

```diff
@@ -0,0 +1,3 @@
+package org.mifos.mobile.cn.data.models.beneficiaries
+
+data class ListBeneficiaries(val name: String, val description: String, val price: String)
```

Comment:
This is representing only a single Beneficiary, convert it to singular.

Reply:
done!